braindecode/datasets/base.py

```diff
@@ -142,6 +144,88 @@ def transform(self, value):
         self._transform = value
 
 
+class SequenceDataset(WindowsDataset):
+    """Dataset of sequences of consecutive windows.
+
+    Parameters
+    ----------
+    windows : mne.Epochs
+        Windows obtained through the application of a windower to a BaseDataset
+        (see `braindecode.datautil.windowers`).
+    description : dict | pandas.Series | None
+        Holds additional info about the windows.
+    transform : callable | None
+        On-the-fly transform applied to a window before it is returned.
+    seq_len : int
+        Number of consecutive windows in a sequence.
+    step_len : int
```

Comment:
Does `step_len` parameter correspond to `stride` in `create_windows` functions? Maybe it would be a better name then.

Reply:
Maybe we could name to `n_windows_seq` and `n_windows_stride`?

Reply:
I would vote for n_windows and n_windows_stride

Reply:
Also fine for me